ipa.go

```diff
@@ -47,6 +49,15 @@ func toFr(fr *Fr, p *Point) {
 	fr.SetBytesLE(bytes[:])
 }
 
+func toFrMultiple(ps []*Point) []Fr {
```

Comment:
Still open to see if returning `[]Fr` is convenient or have a similar API as `toFr(..)` where it could be something like `toFrMultiple(ret []*Fr, ps []*Point)` if that makes things simpler for the caller.

We can try discovering that when we integrate the usage of this in the rest of the repo.  Just mentioning here to be clear this is a tentative API just to benchmark for now.

Reply:
I would expect passing a slice of pointers to be faster, but it's likely dwarfed by the map-to-field conversion. I might be wrong though, it would be interesting to benchmark.